docs/en/token_endpoint.rst

```diff
@@ -205,11 +205,17 @@ UserInfo endpoint to get user attributes.
      - It MUST be an HTTPS URL that uniquely identifies the OP. The client MUST verify that this value matches the called OP.
      - |spid-icon| |cieid-icon|
    * - **sub** 
-     - See `OpenID.Core#SubjectIDTypes`_. It MUST be *pairwise*. 
+     - See `OpenID.Core#SubjectIDTypes`_. It MUST be *pairwise* type. 
```

Comment:
`It MUST be *pairwise* type` è un typo

it must be pairwised oppure it must be of pairwise type

inoltre è misleading, è come se divessimo che il valore di SUB deve essere "pairwise" 
noi dovremmo dire che il valore deve essere di tipo pairwised (con ref a OIDC core)

ok?

Reply:
Per me ok entrambe.

Reply:
Preferisco "of pairwise type"

Reply:
fatto qui 03640c32f64041d5d9388a5367cabf1ac0a47755